common/rpc_service.go

```diff
@@ -37,15 +37,14 @@ import (
 // RPCService the type alias of interface{}
 type RPCService = interface{}
 
-// ReferencedRPCService
-// rpc service interface
+// ReferencedRPCService is the rpc service interface which wraps base Reference method.
+//
+// Reference method refers rpc service id or reference id.
```

Comment:
this line is a method comment, it should better be placed before the method Reerence()?

Reply:
![image](https://user-images.githubusercontent.com/99853473/158748132-0aa21a8c-f44e-461a-b698-81f34e2daa47.png)

It is also better to be consistent with the official interface code style.